index.html

```diff
@@ -8887,8 +8887,292 @@ <h3>Graph Containers</h3>
+  </section>
+
+  <section class="informative"><h3>Representing Singular Values as Arrays</h3>
+  </section>
+
+  <section class="informative"><h3>Representing Lists as Arrays</h3>
+  </section>
+
+  <section class="informative"><h3>Reversing Node Relationships</h3>
+  </section>
+
+  <section class="informative"><h3>Indexing Values</h3>
+  </section>
+
+  <section class="informative"><h3>Normalizing Values as Objects</h3>
+    <p>Sometimes it's useful to compact a document, but keep the
+      node object and value object representations.
+      For this, a term definition can set <code>"@type": "@none"</code>.
+      This causes the <a data-cite="JSON-LD11-API#value-compaction">Value Compaction</a> algorithm to always use the object
+      form of values, although components of that value may be compacted.</p>
+
+  <aside class="example" title="Forcing Object Values">
+    <p>Given the following expanded document:</p>
+    <pre class="expanded" data-transform="updateExample">
+    <!--
+    [{
+      "http://example.com/notype": [
+        {"@value": "string"},
+        {"@value": true},
+        {"@value": false},
+        {"@value": 1},
+        {"@value": 10.0},
+        {"@value": "plain"},
+        {"@value": true, "@type": "http://www.w3.org/2001/XMLSchema#boolean"},
+        {"@value": "english", "@language": "en"},
+        {"@value": "2018-02-17", "@type": "http://www.w3.org/2001/XMLSchema#date"},
+        {"@id": "http://example.com/iri"}
+      ]
+    }]
+    -->
+    </pre>
+    <p>And the following context:</p>
+    <pre class="context" data-transform="updateExample"
+         data-context-for="Forcing Object Values-expanded">
+    <!--
+    {
+      "@context": {
+        "@version": 1.1,
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "notype": {"@id": "http://example.com/notype", "@type": "@none"}
+      }
+    }
+    -->
+    </pre>
+    <p>The compacted version will use string values for the defined terms
+      when the values match the <a>term definition</a>. Note that there is
+      no term defined for "plain", that is created automatically using
+      the <a>vocabulary mapping</a>.
+      Also, the other native values,
+      <code>1</code> and <code>true</code>,
+      can be represented without defining a specific type mapping.</p>
+    <pre class="compacted" data-transform="updateExample"
+         data-result-for="Forcing Object Values-expanded"
+         data-context="Forcing Object Values-context"
+         data-compact>
+    <!--
+    {
+      "@context": {
+        "@version": 1.1,
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "notype": {"@id": "http://example.com/notype", "@type": "@none"}
+      },
+      "notype": [
+        {"@value": "string"},
+        {"@value": true},
+        {"@value": false},
+        {"@value": 1},
+        {"@value": 10.0},
+        {"@value": "plain"},
+        {"@value": true, "@type": "xsd:boolean"},
```

Comment:
shouldn't that be `{"@value": "true", "@type": "xsd:boolean"},` (with quotes around `true`)? I get that both are valid, but I find it odd to use a native value with an explicit datatype...

Reply:
It certainly has the same meaning, but value compaction doesn't have a case for this; If the term were defined with `@type: xsd:boolean` it would certainly collapse down to just `true`.

(To make this a tad less confusing, as we already have `{"@value": true}`, I'm using `{"@value": false, "@type": "http://www.w3.org/2001/XMLSchema#boolean"}` instead).

We could update value compaction for native values (although, it can get tricky for something like `xsd:decimal`), but we'd risk a backwards incompatible change.

It may be simpler to just leave out this particular value for this example.

Reply:
I'm not suggesting that compaction convert `true` to `"true"` in that case... only that the *source* use a string (and so compaction gives that same string back). Unless you think that it is acceptable practice to author JSON-LD with non-strings in the `"@value"` field of a value node... I still find it odd but I can live with that.

> To make this a tad less confusing...

Good idea :) I didn't notice that, but it *can* be confusing :)

Reply:
Compaction should not change native values (such as `true` or `false` to strings), and the `fromRdf` algorithm has an option to use native values when possible. Because of this, when expanding something like `"term": true`, it will always using the value object representation of `"http://example.org/term": [{"@value": true}]`.

We could certainly update the example to use "true" (or "false") instead of the native `true`, which may make this less confusing.